store/nextFreeSlots.js

```diff
@@ -19,10 +19,37 @@ const calendarRangeLoaded = (storeValue, date) => {
   return true;
 }
 
+const setSelectedDateIfInitialNextFreeSlotsLoading = (store) => {
+  const { initialNextFreeSlotsLoading, selectedDate, nextFreeSlots, calendarRange} = store.get();
```

Comment:
vegyük itt ki az appointment-et és a selectedCalendar-t és akkor lentebb nem kell annyiszor store.get().valami-zni

(és esetleg akkor a két return kimenetelű if feltétele is megférhet egymás mellett/alatt)